pwa/src/templates/organizationDetail/OrganizationDetailTemplate.tsx

```diff
@@ -33,9 +33,22 @@ export const OrganizationDetailTemplate: React.FC<OrganizationDetailTemplateProp
           <Heading1>Gemeente Rotterdam</Heading1>
 
           <LeadParagraph>
-            Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Maecenas faucibus mollis interdum. Duis
-            mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Vivamus sagittis
-            lacus vel augue laoreet rutrum faucibus dolor auctor.
+            <div className={styles.quote}>
+              Een stad waar je ruimte ervaart. <br />
+              Bij een organisatie die je vrijheid en verantwoordelijkheid biedt. <br />
+              Voor werk waar je uitdaging voelt. <br />
+              Zodat jij en de stad blijven groeien. <br />
+              Voor en met alle Rotterdammers.
+            </div>
+            <br />
```

Comment:
Same goes for these `<br>` elements. Will we know where to place them? Will we receive `html` from the gateway?